sequencer/src/bin/espresso-dev-node.rs

```diff
@@ -395,6 +492,31 @@ mod tests {
             }
         }
 
+        let dev_node_client: Client<ServerError, SequencerVersion> =
+            Client::new(format!("http://localhost:{dev_node_port}").parse().unwrap());
+        dev_node_client.connect(None).await;
+
+        // Check the dev node api
+        {
+            dev_node_client
+                .get::<DevInfo>("api/dev-info")
+                .send()
+                .await
+                .unwrap();
+
+            dev_node_client
+                .get::<()>("api/mock-contract/freeze/3")
+                .send()
+                .await
+                .unwrap();
```

Comment:
Can we check if the L1 height did freeze or unfreeze?

Reply:
e92c71737d74e7e3db24e017de9bf791f5bfaaac